charts/rekor-tiles/rekor-tiles/templates/deployment.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "rekor-tiles.fullname" . }}
+  labels:
+    {{- include "rekor-tiles.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "rekor-tiles.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "rekor-tiles.labels" . | nindent 8 }}
+        {{- with .Values.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "rekor-tiles.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ template "rekor-tiles.image" .Values.image }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
I would have expected the GCP clients to read this from metadata instead of explicitly having to pass this through; does this enable local testing?

Reply:
I was overcomplicating it by not using GKE. Removed all this.

Reply:
BTW, if you have any suggestions for how to improve the timestamp authority helm chart, we should update that as well. I required GOOGLE_APPLICATION_CREDENTIALS be copied into the image but at the time I didn't really know how else to do it.